tests/functional/encryption/test_intransit_encryption_sanity.py

```diff
@@ -1,74 +1,138 @@
 import logging
 import pytest
-
+import time
 from ocs_ci.ocs.resources.storage_cluster import (
-    in_transit_encryption_verification,
     set_in_transit_encryption,
     get_in_transit_encryption_config_state,
 )
 from ocs_ci.framework.pytest_customization.marks import (
     tier1,
     skipif_ocs_version,
     green_squad,
-    cloud_platform_required,
 )
 from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import create_pods
+from concurrent.futures import ThreadPoolExecutor
 
 log = logging.getLogger(__name__)
 
 
 @green_squad
-@cloud_platform_required
+@skipif_ocs_version("<4.18")
 class TestInTransitEncryptionSanity:
     @pytest.fixture(autouse=True)
     def set_encryption_at_teardown(self, request):
+        """
+        Fixture to restore encryption state and clean up resources after the test.
+        """
+
         def teardown():
-            if config.ENV_DATA.get("in_transit_encryption"):
-                set_in_transit_encryption()
-            else:
-                set_in_transit_encryption(enabled=False)
+            initial_state = config.ENV_DATA.get("in_transit_encryption", False)
```

Comment:
wouldn't this teardown be applicable to other existing or future tests? If so, suggest placing it in conftest.py

Reply:
Done , Moved the fixture in conftest.py